frontend/src/layouts/styles.ts

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export const AppGridContainer = styled.div`
   padding: 0;
   text-align: center;
-  grid: ${props => props.theme.css.headerHeight} auto ${props => props.theme.css.footerHeight} / min-content 1fr;
+  grid: ${props => props.theme.css.headerHeight} 1fr ${props => props.theme.css.footerHeight} / min-content 1fr;
```

Comment:
We need **1fr** here so the content area in the grid takes all available space between header and footer. Auto does not do that but instead takes whatever space needed by the children.

We want it to take all available space so list views don't show "short" scrolls bar when using 5 items per page like shown here (in DEV now)

![image](https://user-images.githubusercontent.com/24322224/135936621-df83d46f-e2cd-4031-97bd-9b4de24f5a98.png)